generator/src/main/java/org/corfudb/generator/operations/SnapshotTxOperation.java

```diff
@@ -12,16 +17,29 @@
 @Slf4j
 public class SnapshotTxOperation extends Operation {
 
+
     public SnapshotTxOperation(State state) {
         super(state);
+        shortName = "TxSnap";
     }
 
     @Override
     public void execute() {
         long trimMark = state.getTrimMark();
         Random rand = new Random();
         long delta = (long) rand.nextInt(10) + 1;
-        state.startSnapshotTx(trimMark + delta);
+
+        // Safety Hack for not having snapshot in the future
+
+        long current_max = state.getRuntime().getSequencerView()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)